windows/src/developer/TIKE/project/Keyman.Developer.UI.Project.UfrmProject.pas

```diff
@@ -286,65 +300,21 @@ procedure TfrmProject.CompileAll;
   end;
 end;
 
-function TfrmProject.ShouldHandleNavigation(URL: string): Boolean;
+procedure TfrmProject.ClearMessages;
 begin
-  Result := False;
-  if Copy(URL, 1, 7) = 'keyman:' then
```

Comment:
Why was `'keyman:'` removed?

Reply:
This is handled in the parent class now.